src/generics/cloud-services.js

```diff
@@ -15,8 +15,9 @@ module.exports = class FilesHelper {
 			)
 
 			return {
-				signedUrl: signedUrl[0],
+				signedUrl: process.env.CLOUD_STORAGE_PROVIDER == 'gcloud' ? signedUrl[0] : signedUrl,
```

Comment:
@nevil-mathew these code it should be handled in the npm 
and this also 
if (['azure', 'gcloud'].includes(process.env.CLOUD_STORAGE_PROVIDER))

Reply:
Yes, but since we're using Sunbird-Ed/client-cloud-services, it would take time. So it was faster to handle here.